tests/std/tests/P0896R4_views_all/test.cpp

```diff
@@ -20,6 +20,33 @@ concept CanViewAll = requires(Rng&& r) {
 // Test a silly precomposed range adaptor pipeline
 constexpr auto pipeline = views::all | views::all | views::all | views::all | views::all | views::all;
 
+// Test weird view type with explicit copy constructor, LWG-3724
+struct weird_string_view {
+    const char* data_  = nullptr;
+    unsigned int size_ = 0u;
+
+    weird_string_view()                                    = default;
+    explicit weird_string_view(const weird_string_view&)   = default;

```

Comment:
`copy_constructible` requires implicit copy construction via `convertible_to<const T&, T>`, so this is a move-only view.